Implementation_Report_3e/index.html

```diff
@@ -199,17 +199,17 @@ <h3>Frame control tests</h3>
                     <table>
                         <tr class="header">
                             <th><a href="https://wpt.fyi/results/png/apng?label=experimental&label=master&aligned">Live Results</a></th>
-                            <th>Chrome 125</th>
-                            <th>Edge 124</th>
+                            <th>Chrome 128</th>
+                            <th>Edge 127</th>
                             <th>Firefox 126</th>
-                            <th>Safari TP 190</th>
+                            <th>Safari TP 196</th>
                         </tr>
                         <tr class="test">
                             <td class="testname">fcTL-acTL-ordering.html </td>
                             <td class="result passes-all"></td>
                             <td class="result passes-all"></td>
                             <td class="result passes-all"></td>
-                            <td class="result passes-all"></td>
+                            <td class="result passes-none"></td>

```

Comment:
Just checking that this change was intentional.
Did it regress? Is it a correction of a previous mistake?

Reply:
"Also fixed an error (one failing test was incorrectly shown in the table as the preceding test, my bad)